### PR TITLE
fix elementwise fuse broadcast

### PR DESCRIPTION
**Pull request by SunNy820828449**
Fix Elementwise + broadcast fusion function.
if elementwise has multi-consumer, set it to be not fused.

### CODE REVIEW

File: cinn/frontend/decomposer/activation.cc [hunk 1]
```diff
@@ -42,8 +43,9 @@ void relu_grad(const Instruction& instr, const DecomposerContext& context) {
   auto dx       = instr->outputs[0];
   auto* builder = context.builder();
 
-  auto zero_var   = builder->ConstScalar<float>(0.f, common::UniqName("zero"));
-  auto bcast_zero = builder->BroadcastTo(zero_var, out->shape, {0});
+  // auto zero_var   = builder->ConstScalar<float>(0.f, common::UniqName("zero"));
```
Comment (zhhsplendid) [inline cinn/frontend/decomposer/activation.cc]: Remove useless comment code?

File: cinn/frontend/decomposer/activation.cc [hunk 2]
```diff
@@ -26,8 +26,9 @@ void relu(const Instruction& instr, const DecomposerContext& context) {
   auto output   = instr->outputs[0];
   auto* builder = context.builder();
 
-  auto zero_var   = builder->ConstScalar<float>(0.f, common::UniqName("zero"));
-  auto bcast_zero = builder->BroadcastTo(zero_var, x->shape, {0});
+  // auto zero_var   = builder->ConstScalar<float>(0.f, common::UniqName("zero"));
```
Comment (zhhsplendid) [inline cinn/frontend/decomposer/activation.cc]: Remove useless comment code?

**Review by zhhsplendid — COMMENTED** [review]
LGTM for other parts

File: cinn/frontend/decomposer/activation.cc (hunk 1 above)
Comment (SunNy820828449) [reply cinn/frontend/decomposer/activation.cc]: done

File: cinn/frontend/decomposer/activation.cc (hunk 2 above)
Comment (SunNy820828449) [reply cinn/frontend/decomposer/activation.cc]: done

File: cinn/hlir/framework/graph.h [hunk 3]
```diff
@@ -96,7 +97,7 @@ class Graph : public cinn::common::Graph {
       }
     }
 
-    std::string GetFuncName() { return "fn_" + group_id; }
+    std::string GetFuncName() { return "fn_" + group_id + unique_id; }
```
Comment (zhhsplendid) [inline cinn/hlir/framework/graph.h:100]: You are using 2 id's to represent a func name. Can we set append `unique_id` to the end of `group_id` where the code set `group_id`? In that case, one id is enough, the code is much beautiful.

**Review by zhhsplendid — COMMENTED** [review]
Since this PR is blocking other people's work, I can give an approval and we change comment in the future PR, let me know if you need it.

**Review by zhhsplendid — APPROVED** [review]
Approve temporarily

File: cinn/hlir/framework/graph.h (hunk 3 above)
Comment (SunNy820828449) [reply cinn/hlir/framework/graph.h:100]: unique_id代表了group序号的唯一性，只用group id会出现名字的重复，应该kernel的注册。

**Pull request merged**